tests/actions.go

```diff
@@ -2007,7 +2089,18 @@ func (oa *operatorActions) checkTiKVConfigUpdated(tc *v1alpha1.TidbCluster, clus
 func (oa *operatorActions) checkPrometheus(clusterInfo *TidbClusterConfig) error {
 	ns := clusterInfo.Namespace
 	tcName := clusterInfo.ClusterName
-	prometheusSvc := fmt.Sprintf("http://%s-prometheus.%s:9090/api/v1/query?query=up", tcName, ns)
+	var prometheusAddr string
+	if oa.fw != nil {
+		localHost, localPort, cancel, err := portforward.ForwardOnePort(oa.fw, ns, fmt.Sprintf("svc/%s-prometheus", tcName), 9090)
+		if err != nil {
+			return err
+		}
+		defer cancel()
+		prometheusAddr = fmt.Sprintf("%s:%d", localHost, localPort)
+	} else {
+		prometheusAddr = fmt.Sprintf("%s-prometheus.%s:9090", tcName, ns)
+	}
```

Comment:
This pattern appears multiple times, can we have a `oa.GetAddr()` (or separately, `oa.GetSvcAddr()` and `oa.GetPodAddr()`) method so that the detail of pod/svc connectivity could be handled in one place and port-forward could be used seamlessly in the new code without extra attention.

Reply:
The implementation detail I mentioned above does not matter, the important thing is that we need a more friendly interface so that the author of test case don't have to care about whether the access to Pod need port-forward or not.

Reply:
I didn't add subroutines because these addresses are used in one place. Another reason is the `oa.fw != nil` check is temporary, after the stability tests are reimplemented in the same framework, this will be simplified, so I think current situation is acceptable. All users should use port-forwarding to access ports of the pod. (or `kubectl exec ...`)

Reply:
OK, LGTM